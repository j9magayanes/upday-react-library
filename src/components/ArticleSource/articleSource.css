.source-meinestadt {
  font-weight: 700;
  color: #000 !important;
  margin-right: 0.3125rem;
  font-family: Roboto Bold,sans-serif;
  font-size: 12px !important;
}

.source{
  font-weight: 700;
  color: #000 !important;
  margin-right: 0.3125rem;
  font-family: 'Inter';
  font-size: 12px !important;
}