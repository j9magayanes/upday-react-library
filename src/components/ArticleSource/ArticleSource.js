import React from 'react';
import './articleSource.css';
import  'upday-web-components/components/articleSource'

export const ArticleSource = ({ className, text }) => {

  return (
   <article-source class={className} text={text}></article-source>
  );
};