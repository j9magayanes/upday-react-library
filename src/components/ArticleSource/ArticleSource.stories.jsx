import { ArgTypes } from '@storybook/blocks'
import React from 'react';
import { ArticleSource } from './ArticleSource';
import { ArticleDetail } from '../ArticleDetail/ArticleDetail';

export default {
  title: 'Form/ArticleSource',
  component: ArticleSource,
};

const Template = (args) => <ArticleSource {...args}  />;

export const Web = Template.bind({});
Web.args = {
    text: "Welt",
    className: "headliner-publisher",
    title: "test"

};

export const Meinestadt = Template.bind({});
Meinestadt.args = {
  text: "Welt",
  className: "headliner-publisher"

};