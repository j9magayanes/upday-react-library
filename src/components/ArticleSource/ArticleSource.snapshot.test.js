import React from 'react';
import renderer from 'react-test-renderer';
import { ArticleSource } from './ArticleSource';

test('renders ArticleSource component correctly', () => {
  const props = {
    className: 'your-class-name',
    text: 'your-text',
  };

  const component = renderer.create(<ArticleSource {...props} />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
