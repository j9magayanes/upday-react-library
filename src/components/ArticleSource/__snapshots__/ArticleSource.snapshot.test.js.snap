// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ArticleSource component correctly 1`] = `
<article-source
  class="your-class-name"
  text="your-text"
/>
`;
